### PR TITLE
Add ranice's summary

### PR DESCRIPTION
**Pull request by raniceyue**

### CODE REVIEW

File: data/coreteam_members.yml [hunk 1]
```diff
@@ -93,4 +93,10 @@ members:
...
     description: |
       a computer science undergraduate with interest in app development, algorithms, research and many other wonky stuff.
-      An avid tea drinker, he also enjoys music and ball games that involve running around. Find him on <a href="https://github.com/sun-yitao">GitHub</a>.
\ No newline at end of file
+      An avid tea drinker, he also enjoys music and ball games that involve running around. Find him on <a href="https://github.com/sun-yitao">GitHub</a>.
+      
+  - name: Ranice
+    description: |
+      a computer science undergrad with a fat interest in web and game development. 
+      When she's not studying, she's ricing her current linux distro, OR playing age of empires OR changing the theme of her website... OR
```
Comment (chrisgzf) [inline data/coreteam_members.yml:101]: y u no link ur website

Comment (raniceyue) [reply data/coreteam_members.yml:101]: my website is in shambles @_@

Comment (raniceyue) [reply data/coreteam_members.yml:101]: just realized this thing is 5 lines long on the website, should i shorten it

Comment (chrisgzf) [reply data/coreteam_members.yml:101]: imo thats fine, mine was 6 lines long initially

**Pull request merged**